releasenotes/notes/rcm-fix-disable-f45787a4e6576f53.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Ensure DD_REMOTE_CONFIGURATION_ENABLED environment variable disables remote config if set to False
```

```suggestion
    Ensure DD_REMOTE_CONFIGURATION_ENABLED environment variable disables remote config if set to False.
```
nit

is it worth mentioning the cases in which DD_REMOTE_CONFIGURATION_ENABLED was not being respected?